### PR TITLE
Added pushover feedstock

### PR DESCRIPTION
**Pull request by krisevans**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/python-pushover/LICENSE
Comment (synapticarbors) [inline recipes/python-pushover/LICENSE:1]: You should remove this file since the license file is already in the source distribution

File: recipes/python-pushover/meta.yaml
Comment (synapticarbors) [inline recipes/python-pushover/meta.yaml:14]: It looks like this has a python entry point. You should add it:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then add a test under `commands`:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

File: recipes/python-pushover/meta.yaml
Comment (synapticarbors) [inline recipes/python-pushover/meta.yaml:30]: `license: GPL-3.0-or-later`

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Review by synapticarbors — APPROVED** [review]

**Comment by synapticarbors** [comment]
I'm happy to merge this once the listed co-maintainer agrees to be listed.

**Comment by dhirschfeld** [comment]
@synapticarbors - I'm happy to be a maintainer for this recipe!

**Pull request merged**
